### PR TITLE
Add paragraph 'Odometry in Global Optimization' to tuning.rst

### PR DESCRIPTION
**Pull request by MichaelGrupp**
Follow https://github.com/googlecartographer/cartographer/pull/1029

### CODE REVIEW

**Comment by MichaelGrupp** [comment]
@gaschler  If you're ok with this you can merge it now that the parameters were added to cartographer.

File: docs/source/tuning.rst [hunk 1]
```diff
@@ -215,3 +215,19 @@ As a next step, we strongly decrease ``global_sampling_ratio`` and ``constraint_
 to compensate for the large number of constraints.
 We then tune for lower latency as explained above until the system reliably works in real time.
 
+Tuning the Weight of Odometry in the Global Optimization
```
Comment (gaschler) [inline docs/source/tuning.rst]: just: Odometry in Global Optimization

File: docs/source/tuning.rst [hunk 2]
```diff
@@ -215,3 +215,19 @@ As a next step, we strongly decrease ``global_sampling_ratio`` and ``constraint_
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a separate odometry source is used as an input for local SLAM (``use_odometry = true``), we can also tune the global SLAM to benefit from this additional information in an optimal way.
```
Comment (gaschler) [inline docs/source/tuning.rst]: drop "in an optimal way"

File: docs/source/tuning.rst [hunk 3]
```diff
@@ -215,3 +215,19 @@ As a next step, we strongly decrease ``global_sampling_ratio`` and ``constraint_
...
+There are in total four parameters that allow us to tune the individual weights of local SLAM and odometry in the optimization:
+
+  .. code-block:: lua
+
+    POSE_GRAPH.optimization_problem.local_slam_pose_translation_weight
+    POSE_GRAPH.optimization_problem.local_slam_pose_rotation_weight
+    POSE_GRAPH.optimization_problem.odometry_translation_weight
+    POSE_GRAPH.optimization_problem.odometry_rotation_weight
+
+We can set these weights depending on how much we trust either local SLAM or the odometry.
```
Comment (gaschler) [inline docs/source/tuning.rst]: Suggestion to replace the last sentence: "By default, odometry is weighted into global optimization similar to local slam (scan matching) poses.
However, odometry from wheel encoders often has a high uncertainty in rotation.
In this case, the rotation weight can be reduced, even down to zero."

Comment (MichaelGrupp) [reply docs/source/tuning.rst]: changed

**Comment by gaschler** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user gaschler. Merge queue now has a length of 1.

**Pull request merged**